backend/tracim_backend/tests/functional/test_reset_password.py

```diff
@@ -36,6 +37,55 @@ def test_api__reset_password_request__ok__nominal_case(self):
         assert headers['Subject'][0] == '[TRACIM] Reset Password Request'
         requests.delete('http://127.0.0.1:8025/api/v1/messages')
 
+    @pytest.mark.email_notification
+    @pytest.mark.unknown_auth
+    def test_api__reset_password_request__ok__unknown_auth(self):
+        # create new user without auth (default is unknown)
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        params = {
+            'email': 'test@test.test',
+            'password': 'mysuperpassword',
+            'profile': 'users',
+            'timezone': 'Europe/Paris',
+            'lang': 'fr',
+            'public_name': 'test user',
+            'email_notification': False,
+        }
+        res = self.testapp.post_json(
+            '/api/v2/users',
+            status=200,
+            params=params,
+        )
+        res = res.json_body
+        assert res['user_id']
+        user_id = res['user_id']
+
+        # make a request of password
+        self.testapp.authorization = None
+        requests.delete('http://127.0.0.1:8025/api/v1/messages')
+        params = {
+            'email': 'test@test.test'
+        }
+        self.testapp.post_json(
+            '/api/v2/auth/password/reset/request',
+            status=204,
+            params=params,
+        )
+        response = requests.get('http://127.0.0.1:8025/api/v1/messages')
+        response = response.json()
+        assert len(response) == 1
+        headers = response[0]['Content']['Headers']
+        assert headers['From'][0] == 'Tracim Notifications <test_user_from+0@localhost>'  # nopep8
+        assert headers['To'][0] == 'test user <test@test.test>'
+        assert headers['Subject'][0] == '[TRACIM] Reset Password Request'
+        requests.delete('http://127.0.0.1:8025/api/v1/messages')
```

Comment:
I think you do not have to manage data in your test - data must be reset between every test case

Reply:
You're right that mailhog data as not properly managed now.

Reply:
two possibilities:

1. if easy, move the `requests.delete` in `tearDown()`
2. if not easy, add a TODO to the code

Reply:
done, cleanup all mailhog test.